plugins/MultiSites/Columns/Metrics/EcommerceOnlyEvolutionMetric.php

```diff
@@ -36,13 +36,13 @@ public function compute(Row $row)
 
         // if the site this is for doesn't support ecommerce & this is for the revenue_evolution column,
         // we don't add the new column
-        if (($currentValue === false
-            || !$this->isRevenueEvolution)
-            && !Site::isEcommerceEnabledFor($row->getColumn('label'))
-        ) {
-            $row->deleteColumn($columnName);
+        if ($currentValue === false || !$this->isRevenueEvolution) {
+            $idSite = $row->getMetadata('idsite');
```

Comment:
Label might be already replaced with the Site name. We need to use the idsite metadata instead which always works